docs/components/modeler/web-modeler/deploy-process-application.md

```diff
@@ -8,13 +8,14 @@ import DeployImg from './img/process-applications/define-stages-deploy.png'
 import RunProcessApplicationImg from './img/process-applications/run-process-application.png'
 import DeployErrorImg from './img/process-applications/deploy-error.png'
 
-You can deploy your process application to a cluster, and run your process application to test, debug, and observe how it performs in a live environment.
+You can deploy your process application to a cluster, and play or run your process application to test, debug, and observe how it performs in a development or live environment.
```

Comment:
This is starting to get really long, so let's see how it can be fixed. Here's the user journey:

A user is developing their process. They `play` their process to test, debug, and observe how it performs in a development environment.
After development, they `deploy` to a test, stage, or prod environment to run automated tests or have it available for a testing environment. For manual testing or as needed (it's more edge case-y), they can manually `run` their process.

I don't see a good use case for testing, debugging, or observing a process's performance in a live (production) environment.